syntax = "proto3";

package pwm.proto;

message EncryptionHeader {
	string key_salt = 1;
	uint64 key_size_octets = 2;
	int64 key_work_factor = 3;
	int64 key_parallelization_factor = 4;
}
